Sources/WhisperKit/Core/WhisperKit.swift

```diff
@@ -266,12 +266,87 @@ open class WhisperKit {
             throw error
         }
     }
+    public static func modelLocation(
```

Comment:
Should add a new line above this for code style. Will want to add docc for all of the functions in this file eventually but not a blocker.